src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -0,0 +1,242 @@
+
+    MessageWall wall = new MessageWall().open(user.getUsername());
+    MiniEditorComponentObject mini = wall.triggerMessageArea();
+    String message = PageContent.MESSAGE_WALL_MESSAGE_PREFIX + wall.getTimeStamp();
+    String title = PageContent.MESSAGE_WALL_TITLE_PREFIX + wall.getTimeStamp();
+    mini.switchAndWrite(message);
+    wall.setTitle(title);
+    wall.submit();
+    wall.verifyMessageText(title, message, user.getUsername());
+    wall.triggerEditMessageArea();
+    String messageEdit = PageContent.MESSAGE_WALL_MESSAGE_EDIT_PREFIX + wall.getTimeStamp();
+    mini.switchAndEditMessageWall(messageEdit);
+    wall.submitEdition();
+    wall.verifyMessageEditText(title, messageEdit, user.getUsername());
+
+    String newName = "NewUser Nąmę" + timestamp;
+    SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage(driver)
+        .open()
+        .fillFormData(newName, newName, user.getPassword())
+        .agreeToTermsAndConditions()
+        .submitChange();
+    new ConfirmationModalPage(driver).accept();
+    Assertion
+        .assertEquals(renameUserPage.getSuccessBoxMessage(), "Rename process is in progress. The "
+                                                             + "rest will be done in background. You will be notified via e-mail when it is completed.");
+    wall.open(newName);
+    String encodedUrl = renameUserPage.encodeToURL(newName);
+
+    String expectedWallUrl = String.format("%s/wiki/Message_Wall:%s", urlBuilder.getUrlForWiki(),
+                                           encodedUrl);
+
+    Assertion.assertEquals(driver.getCurrentUrl(), expectedWallUrl);
+    wall.verifyMessageEditTextRenameDone(title, message, newName);
+
+  }
+
+  @Test(groups = {"renameTool_01"})
+  public void NewUserCreateBlogPostAndRenameDone() {
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    SignUpUser
+        user =
+        new SignUpUser("QARename Usęr" + timestamp, credentials.email, "aaaa",
+                       LocalDate.of(1993, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    new WikiBasePageObject().loginAs(user.getUsername(), user.getPassword(), wikiURL);
+
+    String blogTitle = PageContent.BLOG_POST_NAME_PREFIX + DateTime.now().getMillis();
+    String blogContent = PageContent.BLOG_CONTENT + DateTime.now().getMillis();
+
+    UserProfilePage userProfile = new UserProfilePage().open(user.getUsername());
+    userProfile.clickOnBlogTab();
+    SpecialCreatePage createBlogPage = userProfile.clickOnCreateBlogPost();
+    VisualEditModePageObject visualEditMode = createBlogPage.populateTitleField(blogTitle);
+    visualEditMode.addContent(blogContent);
+    BlogPage blogPage = visualEditMode.submitBlog();
+    blogPage.getBlogTitle();
+    blogPage.verifyContent(blogContent);
+
+    UserProfilePage userProfilePage = new UserProfilePage()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "userProfilePage" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)